addons/account/data/security_mail_templates.xml

```diff
@@ -0,0 +1,82 @@
+        </template>
+
+        <template id="partner_bank_account_changed_template" >
+            <p>Dear <t t-esc="user_name" />,<br/><br/>
+                You receive this email because we would like to be sure that you are the person who changed the bank account 
+                of partner <t t-esc="partner.name" />
+                <t t-if="partner.company_id">
+                    in <t t-esc="partner.company_id.name" />
+                </t>
+                at <t t-esc="timestamp" /> (<t t-esc="', '.join(operations)"/>; you can check for more informations on the chatter of the partner).
+            </p>
+            <t t-call="account.security_scammer_generic" />
+            <p>If you think you are victim of a scam:
+                <ul>
+                    <li>Take security measures (change password, lock screen when away...) to avoid this issue in the future.</li>
+                    <li>Log in
+                        <t t-if="partner.company_id">
+                            company <t t-esc="partner.company_id.name" />
+                        </t>
+                        and have a look at partner <t t-esc="partner.name" />. If the bank account for this vendor is incorrect, fix it. Then check no bills were created (or worse, paid) between
+                        <t t-esc="timestamp" /> and now.</li>
+                </ul>
+            </p>
+            <p>
+                Regards,
+            </p>
+            <p>Odoo</p>
+        </template>
+
+        <template id="user_email_changed_template">
+            <p>Dear <t t-esc="changed_user.name" />,<br/><br/>
+                Your email was modified on <t t-esc="timestamp" /> by user <t t-esc="user.name" />.
+                <t t-esc="old_email"/> --> <t t-esc="changed_user.email" />.
+            </p>
+            <p>
+                Regards,
+            </p>
+            <p>Odoo</p>
+        </template>
+
+        <template id="document_layout_changed_template">
+            <p>Dear <t t-esc="user_name"/>,<br/><br/>
+            You receive this email because we would like to be sure that you are the person who changed the document 
+            template of company <t t-esc="company_name" /> at <t t-esc="timestamp" />.
+            </p>
+            <t t-call="account.security_scammer_generic" />
+            <p>If you think you are victim of a scam:
+                <ul>
+                    <li>Take security measures (change password, lock screen when away...) to avoid this issue in the future.</li>
+                    <li>Log in company <t t-esc="company_name" /> and have a look on the document layout in the general settings.
+                        Check if everything is correct with a particular focus on bank account.</li>
+                    <li>Verify no sales invoices were sent  between <t t-esc="timestamp" /> and now. Contact your customers if necessary.</li>
```

Comment:
It is, but with variations since the suggestions are tailored to to the action that was performed.

There wasn't any suggestion for the email changing user case, I added one.